integration_tests/mockApis/visitScheduler.ts

```diff
@@ -16,4 +19,78 @@ export default {
       },
     })
   },
+  stubGetPrison: (prisonCode: string): SuperAgentRequest => {
+    return stubFor({
+      request: {
+        method: 'GET',
+        url: `/visitScheduler/admin/prisons/prison/${prisonCode}`,
+      },
+      response: {
+        status: 200,
+        headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+        jsonBody: TestData.prison(),
+      },
+    })
+  },
+  stubCreatePrison: (prison: Prison): SuperAgentRequest => {
+    return stubFor({
+      request: {
+        method: 'POST',
+        url: '/visitScheduler/admin/prisons/prison',
+        bodyPatterns: [
+          {
+            equalToJson: {
+              active: prison.active,
+              code: prison.code,
+              excludeDates: prison.excludeDates,
+            },
+          },
+        ],
+      },
+      response: {
+        status: 201,
+        headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+        jsonBody: prison,
+      },
+    })
+  },
+  stubActivatePrison: (prisonCode: string): SuperAgentRequest => {
+    return stubFor({
+      request: {
+        method: 'PUT',
+        url: `/visitScheduler/admin/prisons/prison/${prisonCode}/activate`,
+      },
+      response: {
+        status: 200,
+        headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+        jsonBody: { ...TestData.prison, active: true },
```

Comment:
Should be:
```
jsonBody: { ...TestData.prison(), active: true },
```
or
```
jsonBody: TestData.prison({ active: true }),
```
because `.prison` is a function not a property. It happens to be working because the code only checks the `active` property and not the whole response.